lib/hexpm/billing/local.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Hexpm.Billing.Local do
```

Comment:
wdyt about making this a tiny bit more real by loading/dumping data from tmp, similarly to Hexpm.Store.Local? This way dashboard UI would be more usable, currency it looks broken (obviously does not persist stuff). I'm happy to tackle this after this PR lands too. But yeah, maybe extra complexity isn't worth it.

Reply:
I think it would be hard to make it useful, as long as nothing crashes and does not "look" broken I am happy with it. If you want to improve things then please go ahead of course.